back/app/serializers/web_api/v1/custom_field_serializer.rb

```diff
@@ -45,7 +49,7 @@ class WebApi::V1::CustomFieldSerializer < WebApi::V1::BaseSerializer
     :linear_scale_label_9_multiloc,
     :linear_scale_label_10_multiloc,
     :linear_scale_label_11_multiloc,
-    if: proc { |object, _params| object.linear_scale? || object.supports_matrix_statements? }
+    if: proc { |object, _params| object.linear_scale? || object.sentiment_linear_scale? || object.supports_matrix_statements? }
```

Comment:
Might be cleaner to have an attribute on the model `supports_linear_scale_labels?`